components/nexusoperations/config.go

```diff
@@ -96,26 +95,19 @@ ScheduleNexusOperation commands with a "nexus_header" field that exceeds this li
 Uses Go's len() function on header keys and values to determine the total size.`,
 )
 
-var DisallowedOperationHeaders = dynamicconfig.NewNamespaceTypedSettingWithConverter(
+// defaultDisallowedOperationHeaders - set in the convert function below due to a limitation in the dynamic config framework.
```

Comment:
let's add a TODO somewhere to restore this after 7052?

Reply:
```suggestion
// defaultDisallowedOperationHeaders - set in the convert function below due to a limitation in the dynamic config framework.
// TODO restore after an upgrade to Go 1.24 and merging #7052.
```